docs/tutorials/basic/reshape_transpose.md

```diff
@@ -1,7 +1,6 @@
 ## Difference between reshape and transpose operators
 
 What does it mean if MXNet gives you an error like the this?
-
```

Comment:
Hi @NRauschmayr , There's also another way of pushing an empty commit in case you need to re-trigger the CI. 
You can try ```git commit --allow-empty -m "<commit-message>"``` :)